### DISCUSSION TITLE
Installing on windows

### DISCUSSION THREAD

**Discussion by ENGO150** [nlohmann/json#3310] [Q&A] [answered]
Hi.

I am developing my project on Linux, where I installed this library using `cmake..`, `make` and `make install`... That seems not possible on Windows... I searched many discussions, but found nothing... I also tried nuget package, but still wasn't working...
Is there anyone using this on Windows? thx

**Comment by nlohmann** [comment, marked as answer]
I am not sure how the Windows equivalent of `make install` looks like on Windows. I guess the best approach would be to use a package manager, see https://json.nlohmann.me/integration/package_managers/.

**Reply by ENGO150** [reply]
I tried to implement CPM, but I'm still getting `Cannot open include file: 'nlohmann/json.hpp': No such file or directory `...

[full run](https://github.com/ENGO150/ClearTemp/runs/5005037046?check_suite_focus=true)

**Reply by nlohmann** [reply]
Do you have a `target_link_libraries` call somewhere?

**Reply by Tom-Evers** [reply]
It almost looks like you forgot to include the directory that contains the `nlohmann` folder

**Reply by nlohmann** [reply]
@ENGO150 Do you need further assistance with this issue?

**Reply by ENGO150** [reply]
Oh, I'm sorry.. No, problems with building are fixed. Thank you so much.
